### PR TITLE
Clean up claims

### PR DESCRIPTION
**Pull request by vicancy**
Sticky session does not depend on signature anymore

### CODE REVIEW

File: test/Microsoft.Azure.SignalR.Tests/ServiceEndpointProviderFacts.cs [hunk 1]
```diff
@@ -106,8 +106,8 @@ internal void GetClientEndpointWithAppName(IServiceEndpointProvider provider, st
             Assert.Equal(expected, actual);
         }
 
-        [Fact]
-        internal void GenerateMutlipleAccessTokenShouldBeUnique()
+        [Fact(Skip = "Access token does not need to be unique")]
+        internal void GenerateMultipleAccessTokenShouldBeUnique()
```
Comment (JialinXin) [inline test/Microsoft.Azure.SignalR.Tests/ServiceEndpointProviderFacts.cs]: Why is this different in AspNet and Core?

Comment (vicancy) [reply test/Microsoft.Azure.SignalR.Tests/ServiceEndpointProviderFacts.cs]: No its not, they are all skipped.

Comment (JialinXin) [reply test/Microsoft.Azure.SignalR.Tests/ServiceEndpointProviderFacts.cs]: Yes. Didn't see it.

File: src/Microsoft.Azure.SignalR.Common/Utilities/AuthenticationHelper.cs [hunk 2]
```diff
@@ -25,15 +25,14 @@ internal static class AuthenticationHelper
...
-            var requestIdClaim = new Claim[] { new Claim(Constants.ClaimType.Id, requestId ?? GenerateRequestId()) };
-            var claimsWithRequestId = claims == null ? requestIdClaim : claims.Concat(requestIdClaim);
-            var subject = new ClaimsIdentity(claimsWithRequestId);
-            return GenerateJwtBearer(issuer, audience, subject, expires, signingKey);
+            var subject = claims == null ? null : new ClaimsIdentity(claims);
+            return GenerateJwtBearer(issuer, audience, subject, expires, signingKey, issuedAt, notBefore);
         }
 
-        public static string GenerateAccessToken(string signingKey, string audience, IEnumerable<Claim> claims, TimeSpan lifetime, string requestId = null)
+        public static string GenerateAccessToken(string signingKey, string audience, IEnumerable<Claim> claims, TimeSpan lifetime, DateTime? issueAt = null)
```
Comment (realwanpengli) [inline src/Microsoft.Azure.SignalR.Common/Utilities/AuthenticationHelper.cs]: no one pass the `issueAt `?

Comment (vicancy) [reply src/Microsoft.Azure.SignalR.Common/Utilities/AuthenticationHelper.cs]: nice catch

**Pull request merged**